Index: Core/MOOSGenLib/MOOSSerialPort.h
===================================================================
--- Core/MOOSGenLib/MOOSSerialPort.h	(revision 4237)
+++ Core/MOOSGenLib/MOOSSerialPort.h	(working copy)
@@ -33,8 +33,8 @@
 
 /*! \file MOOSSerialPort.h */
 
-#if !defined(MOOSERIALPORTH)
-#define MOOSSERIALPORTH
+#if !defined(MOOSSERIALPORTH)
+#define MOOSSERIALPORTH
 
 #if _MSC_VER > 1000
 #pragma once
